pkgs/applications/editors/atom/default.nix

```diff
@@ -1,6 +1,7 @@
 { stdenv, fetchurl, buildEnv, makeDesktopItem, makeWrapper, zlib, glib, alsaLib
 , dbus, gtk, atk, pango, freetype, fontconfig, libgnome_keyring3, gdk_pixbuf
 , gvfs, cairo, cups, expat, libgpgerror, nspr, gconf, nss, xorg, libcap, systemd
+, version ? "1.6.0"
```

Comment:
I think it would be nicer if we did this similar to https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/build-managers/gradle/default.nix.
Create a generic builder and then pass in version, url and sha256 into that.